Gems/Atom/RHI/DX12/CMakeLists.txt

```diff
@@ -13,3 +13,5 @@ o3de_restricted_path(${gem_json} gem_restricted_path gem_parent_relative_path)
 ly_add_external_target_path(${CMAKE_CURRENT_LIST_DIR}/3rdParty)
 
 add_subdirectory(Code)
+
+ly_create_alias(NAME Atom_RHI_DX12.HeadlessServers NAMESPACE Gem)
```

Comment:
Just curious - why do we need this? Wouldn't a headless server always use a null RHI?

Reply:
It ended up during my testing that this was needed because monolithic builds do not dynamically load the Atom RHIs, they are all linked together. So this wasn't necessary for profile builds, but monolithic builds were pulling in Xcb* dependencies (see response to the other question below)